low-level/framework/utils/filestore.py

```diff
@@ -117,15 +119,15 @@ def _load_json_file(self, key):
             except:
                 value = fh.read()
         except IOError as err:
-            logger.warning("I/O error[{0}] while loading data from file {1}): {2}"\
+            logger.warn("I/O error[{0}] while loading data from file {1}): {2}"\
```

Comment:
why we changed the warning to warn?
As I understand logging.warn has been deprecated since Python 3.3
https://stackoverflow.com/a/15655674

Reply:
Yes, `warn` is depreciated in native python logger. But the logger object here is coming from a custom CORTX Log class which has `warn` method and not `warning`. 
Ref: https://github.com/Seagate/cortx-utils/blob/main/py-utils/src/utils/log.py

Reply:
Okay